pyspark/test/bigdl/test_simple_integration.py

```diff
@@ -544,9 +545,14 @@ def test_predict(self):
             assert_allclose(p_with_batch[i], ground_label[i], atol=1e-6, rtol=0)
 
         predict_class = model.predict_class(predict_data)
-        predict_labels = predict_class.take(6)
-        for i in range(0, total_length):
-            assert predict_labels[i] == 1
+        if isinstance(predict_class, RDD):
+            for sample in predict_class:
```

Comment:
I am confusing why it's working. if it's RDD, we should execute action first before iterate through it, no ?

Reply:
I searched ways to iterate a RDD on the Internet.. I think I could follow this example. I will change my code. 

# create Spark context with Spark configuration
  conf = SparkConf().setAppName("Print Contents of RDD - Python")
  sc = SparkContext(conf=conf)
 
  # read input text file to RDD
  rdd = sc.textFile("data/rdd/input/file1.txt")
 
  # collect the RDD to a list
  list_elements = rdd.collect()
 
  # print the list
  for element in list_elements:
    print(element)